lib/adroll/service.rb

```diff
@@ -57,7 +57,7 @@ def make_api_call(request_method, request_uri, query_params)
 
       if request_method == :get
         perform_get(request_method, request_uri, query_params)
-      elsif request_uri.include?('/ad/create')
+      elsif request_uri.include?('/ad/create?')
```

Comment:
At this step in the process the request_uri will only contain the service name and the service method (`/service/service_method`); it won't contain a `?`. Since `create_templated_web_ads` doesn't pass any images to the api, I'm not sure it will play nice with using a multi post, so we'll have to test that out. Come see me on how to test your code in the console.